src/usr/lib/rsetup/cli/u-boot-menu.sh

```diff
@@ -136,28 +136,46 @@ rebuild_overlays() {
 enable_overlays() {
     __parameter_count_at_least_check 1 "$@"
 
-    local enable_overlays_list i
+    local enable_overlays_list i directory_name file_name have_illegal_parameters=false
```

Comment:
```suggestion
    local enable_overlays_list i dir_name file_name input_check=true
```
名字可以稍微短一点
为了适应input_check的语义也把默认flag改成true